lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -19,11 +19,9 @@
 -- Copyright: © 2018-2019 IOHK
 -- License: MIT
 --
--- This module contains the core primitive of a Wallet. This is roughly a
--- Haskell translation of the [Formal Specification for a Cardano Wallet](https://github.com/input-output-hk/cardano-wallet/blob/master/specifications/wallet/formal-specification-for-a-cardano-wallet.pdf)
+-- Contains primitive wallet types needed for @Cardano.Wallet.Primitive.Model@.
```

Comment:
Well, this sentence isn't really true. Primitive types are used... basically everywhere. And, there's no more import-cycle going on since we split the bridge to another package.

Reply:
Reverted.